pymbar/fes.py

```diff
@@ -2415,16 +2416,20 @@ def ddexpf(x, k_inner):
                             h[i, j] += pE / pF[k]
 
         elif spline_weights == "unbiasedstate":
+
+            def ddexpf(x, index_i, index_j):
```

Comment:
I think the function is supposed to be inside of the loop, as the function (if I recall!) need to be redefined with the loop.  I remember struggling with the right way to do this. Now, there's probably a better way to do it than I did, but I will need to go back and make sure the behavior hasn't changed.   Before release at least.

Reply:
Ah, I see what you did.  OK, I think that will work, I will double check to make sure.  It probably improves performance . . .